README.md

```diff
@@ -94,128 +94,118 @@ $ yarn workspace [name-of-package] add [name-of-npm-package]
+Once you've got the data file and installed Docker and have the PostgreSQL image running as described above, you'll want to load the data onto the image and persist it to the volume.
 
-[//]: # ()
-[//]: # (*Note that due to the size of the data file, some of these steps might take a while — thus, while developing it might make sense to use a smaller subset of the data.*)
+*Note that due to the size of the data file, some of these steps might take a while — thus, while developing it might make sense to use a smaller subset of the data.*
 
-[//]: # ()
-[//]: # (1. Connect to the Docker image in with your preferred DB client)
+1. Connect to the Docker image in with your preferred DB client
 
-[//]: # ()
-[//]: # (2. Run the following query to create the table / schema:)
+2. Run the server at least once (no dump mode) to get the db and table definitions created
 
-[//]: # ()
-[//]: # (```sql)
-
-[//]: # (CREATE TABLE events &#40;)
-
-[//]: # (    id              INT PRIMARY KEY,)
-
-[//]: # (    block_number    INT,)
-
-[//]: # (    nft_name        TEXT,)
-
-[//]: # (    contract_hash   VARCHAR&#40;66&#41;,)
-
-[//]: # (    txn_hash        VARCHAR&#40;66&#41;,)
-
-[//]: # (    txn_type        VARCHAR&#40;10&#41;,)
-
-[//]: # (    gas             BIGINT,)
-
-[//]: # (    value           VARCHAR&#40;78&#41;,)
-
-[//]: # (    from_hash       VARCHAR&#40;42&#41;,)
-
-[//]: # (    to_hash         VARCHAR&#40;42&#41;,)
-
-[//]: # (    token_id        VARCHAR&#40;78&#41;,)
-
-[//]: # (    timestamp       DATE,)
-
-[//]: # (    createdAt       DATE,)
-
-[//]: # (    updatedAt       DATE)
-
-[//]: # (&#41;;)
-
-[//]: # (```)
-
-[//]: # ()
-[//]: # (3. Then, you'll need to get move the `.csv` file form your host machine to the container.)
-
-[//]: # ()
-[//]: # (    * You can get the container_id of the container by running `docker ps` on the command line)
-
-[//]: # ()
-[//]: # (```bash)
-
-[//]: # ($ docker cp /path/events.csv container_id:/events.csv)
-
-[//]: # (```)
-
-[//]: # ()
-[//]: # (4. Now, populate the Postgres DB by running the following query which copies the `.csv` data into the DB volume)
+```sql
+CREATE TABLE dumped_blocks (
+    id              SERIAL PRIMARY KEY,
+    number          VARCHAR(10),
+    timestamp       DATE,
+    createdAt       DATE,
+    updatedAt       DATE
+);
+```
+```sql
+CREATE TABLE events (
+    id              INT PRIMARY KEY,
+    block_number    VARCHAR(10),
+    nft_name        TEXT,
+    contract_hash   VARCHAR(66),
+    txn_hash        VARCHAR(66),
+    txn_type        VARCHAR(10),
+    gas             VARCHAR(32),
+    value           VARCHAR(78),
+    from_hash       VARCHAR(42),
+    to_hash         VARCHAR(42),
+    token_id        VARCHAR(78),
+    timestamp       DATE,
+    createdAt       DATE,
+    updatedAt       DATE
```

Comment:
let's delete these, they're just sequelize artifacts and postgres doesn't play nice with the camelCasing.

Reply:
done